matyhtf/framework/CVE-2021-43676.yaml

```diff
@@ -0,0 +1,8 @@
+title: Path manipulation
+link: https://github.com/advisories/GHSA-mh9j-v6mq-pfch
+cve: CVE-2021-43676
+reference: composer://matyhtf/framework
+branches:
+    '3.0':
+        time: null
```

Comment:
this should have the date set to the time at which the fix was merged

Reply:
As I said, there is not fix yet.

Reply:
if there were no fix yet, `<=3.0.5` as range would have been wrong too